resources/lang/vi/common.php

```diff
@@ -73,11 +74,12 @@
         'minute_short_unit' => 'phút',
         'minutes' => ':count phút',
         'months' => ':count tháng',
-        'notifications' => '',
-        'plus_others' => '',
+        'notifications' => ':count_delimited thông báo|:count_delimited thông báo',
+        'plus_others' => '+ :count_delimited khác!+ :count_delimited khác!',
```

Comment:
missing separator (`|`). I've updated it in crowdin so maybe run another update.

Reply:
actually Vietnamese don't even need this as there's no change for plural things :thinking:

Reply:
Does that actually cause a runtime issue?

Reply:
the extra versions are ignored. Although without the separator it'll just look weird...